site/controllers/city.js

```diff
@@ -0,0 +1,24 @@
+const City = require("../models/city");
+
+exports.getCity = async (req, res, next) => {
+  const city = await City.findAll();
+  console.log(req.city);
+  res.send(city);
+};
+
+exports.getAddCity = (req, res, next) => {
+  res.render("addCity.jsx");
+  //name of the file may vary//
+};
+
+exports.postAddCity = async (req, res, next) => {
+    console.log(req.body);
+    const cityName = req.body.cityName;
+    const city = new City ({cityId, cityName})
```

Comment:
you don't need cityId here since it will be auto-generated and you can use build instead of new